src/periphery/VPM.cpp

```diff
@@ -163,6 +163,25 @@ static constexpr T demodulo(T val, T limit)
     return val == 0 ? limit : val;
 }
 
+static std::string toVPitchString(uint8_t vpitch, uint8_t mode)
+{
+    vpitch = demodulo(vpitch, uint8_t{16});
+
+    if(mode >= 4) // 8-bit witdh
+    {
+        auto y = (vpitch >> 2) & 0x3;
+        auto b = vpitch & 0x3;
+        return "{" + std::to_string(y) + ", " + std::to_string(b) + "}";
```

Comment:
Is there a particular reason why you use braces `{}` instead of parantheses `()`, like we do e.g. for the VPM offset/address

Reply:
I followed the VC4 reference guide. I'll fix it.